modules/standard/Spawn.chpl

```diff
@@ -45,41 +108,74 @@ module Spawn {
   private extern proc qio_spawn_free_str(str: c_string);
 
 
-  /* A subprocess */
+  /* 
+     This record represents a subprocess.
+   */
   record subprocess {
+    /* The kind of a subprocess is used to create the types
+       for any channels that are necessary. */
     param kind:iokind;
+    /* As with kind, this value is used to create the types
+       for any channels that are necessary. */
     param locking:bool;
 
+    pragma "no doc"
     var home:locale;
 
+    /* The Process ID number of the spawned process */
```

Comment:
We could no-doc these things, that would be fine with me.
They aren't C concepts, they are operating systems (linux/POSIX/whatever) concepts.
Part of the purpose of this module is to interact with the operating system, so I think
it makes some sense that they can be exposed.

That said, I could explain them better or refer the reader somewhere else for details.